src/theme/themes/plh_kids_tz/_overrides.scss

```diff
@@ -18,8 +18,16 @@ body[data-theme="plh_kids_tz"] {
   }
   // Button
   plh-button {
+    ion-button {
+      &.information[data-variant~="default"] {
+        --box-shadow: 0px 1.5px 8px rgba(36, 120, 199, 0.5);
```

Comment:
nit(blocking): can we use a theme colour for these rather than a hardcoded value? E.g. `var(--color-primary-600)`